### DISCUSSION TITLE
Visualizzare IP camera PPoE collegate a NVR Hikvision

### DISCUSSION THREAD

**Discussion by maicol07** [indomus/forum#342] [HUB: Home Assistant]
Ciao a tutti,
ho un NVR Hikvision a cui ho collegato delle telecamere PPoE. L'NVR si collega alle telecamere utilizzando il protocollo ONVIF. Le telecamere non sono visibili dalla mia rete locale poichè connesse direttamente all'NVR.
Vorrei sapere se c'è un modo per vedere i flussi video direttamente da Home Assistant così da abbandonare il software "legacy" di HikVision (_sul serio, com'è possibile che nel 2021 si ha un'interfaccia web che funziona solo con Internet Explorer??_).
Ho provato vari indirizzi ONVIF e RTSP ma non sembrano avere successo...
Sapete indirizzarmi sulla soluzione?
Grazie

**Comment by KC-inDomus** [comment]
ciao Marco, hai dimenticato di citare la cosa più importante per farti aiutare, ovvero i modelli di IP Cam e di NVR in uso :)

**Reply by maicol07** [reply]
Il modello di NVR è questo: 36C08-POE-PNP
Le telecamere non so

**Comment by Macgiver1976** [comment]
Ciao, la scheda di rete che gestisce le telecamere connesse in PPoE fa routing verso la scheda di rete con cui controlli l'NVR, quindi dalla rete connessa allo switch non puoi raggiungere le telecamere PPoE.

**Reply by maicol07** [reply]
L'NVR non dovrebbe dare un indirizzo per visualizzarle tutte? Non volevo vedere ogni singola telecamera tramite ognuna, ma tutte direttamente dall'NVR. Con la sua app si vedono, quindi un indirizzo dovrebbe esserci, in modo così da connettermi tramite Home Assistant o altri player

**Reply by Macgiver1976** [reply]
Non proprio: dall'indirizzo ip che vedi le telecamere, in realtà vedi il flusso RTMP che le telecamere producono verso l'NVR. Dovresti fare qualche ricerca su google e vedere se riesci a trovare il flusso che produce l'nvr per ciascuna telecamera e poi lo puoi gestire. 
In alternativa, dovresti collegarti col pc ad una porta PPoE dove sono collegate le telecamere, l'NVR ti rilascia un DHCP e da quella rete sicuramente le trovi
Ciao!

**Reply by Macgiver1976** [reply]
Ho visto che il tuo NVR è un Herospeed, non conosco quel modello, ma in linea di massima funzionano tutti così

**Reply by maicol07** [reply]
Ho cercato ma non ho trovato nulla di funzionante online 😞
